docs/components/IDGeneration.md

```diff
@@ -22,6 +22,7 @@ The two included in the package are:
 
```

Comment:
You are right, it should say three. I will update.

What do you mean clarify what is enumerated? There are no additional requirements other than what is listed, which basically includes Postgres and defining a shard epoch time. All of the exact requirements are in the Using section for this feature, which is in the ShardingAModel.md file. If thats not what you mean, let me know!